src/sqlittaaja/diff_html.py

```diff
@@ -0,0 +1,29 @@
+import difflib
+
+make_table = difflib.HtmlDiff(wrapcolumn=50).make_table
+
+
+def generate_html_table(
```

Comment:
Put this into `output.py`. Don't create a new file for every function.

Reply:
Also the indentation level we use for HTML is 2 but `make_table` seems to use 4?